src/gui/editors/TimeLineWidget.cpp

```diff
@@ -22,23 +22,29 @@
  *
  */
 
+#include "TimeLineWidget.h"
+
+#include <cmath>
 
 #include <QDomElement>
-#include <QTimer>
 #include <QMouseEvent>
 #include <QPainter>
+#include <QTimer>
 #include <QToolBar>
 
-
-#include "TimeLineWidget.h"
 #include "embed.h"
-#include "NStateButton.h"
 #include "GuiApplication.h"
+#include "NStateButton.h"
 #include "TextFloat.h"
 
 namespace lmms::gui
 {
 
+namespace
+{
+	static const int MIN_BAR_LABEL_DISTANCE = 35;
```

Comment:
```suggestion
	constexpr int MIN_BAR_LABEL_DISTANCE = 35;
```
`static` isn't needed within an anonymous namespace, and it can be made `constexpr` instead of just `const`.